apps/console/src/extensions/components/groups/wizard/create-group-wizard.tsx

```diff
@@ -320,7 +322,6 @@ export const CreateGroupWizard: FunctionComponent<CreateGroupProps> = (props: Cr
      * Handles wizard step submit.
      *
      * @param values - Forms values to be stored in state.
-     * @param {WizardStepsFormTypes} formType - Type of the form.
```

Comment:
Shouldn't we keep the comment for the `formType` param?

Reply:
Fixed with aa6b04cb1d9612a46456994d7aeccf49e7b02602